engines/dradis-api/app/views/dradis/ce/api/v1/evidence/_evidence.json.jbuilder

```diff
@@ -4,3 +4,5 @@ json.issue do |json|
   json.title evidence.issue.title
   json.url dradis_api.issue_url(evidence.issue_id)
 end
+json.comments_count evidence.comments_count
```

Comment:
Use a partial for those 2 lines used in issues, evidence and notes.

I am thinking if the need of this partial could be automatically detected (something like `respond_to?(:comments)`) and then automatically added. Would be nice, but how? 🤔